### ISSUE TITLE
[BUG] Nested table reporting an recommendation which seems to be met by the HTML code.

### ISSUE THREAD

**Issue by taniwallach** [IBMa/equal-access#320]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [x ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

> Platform: Chrome plugin on Debian 10 Linux.

> Describe in detail the issue you're having.

Tool reports a recommendation for requirement `1.3.1 Info and Relationships` that `A complex data table should have a 'summary' or an 'aria-describedby' that references an overview of the table` even though the outer table has `aria-labelledby="req_info_summary1"` set.

## Steps to reproduce the issue

1. Scan the HTML code:
```
<html lang="en-US">
<head><title>Issue in nested table</title></head>
<body>
<main>
<div style="text-align:left">
 <h1>Error</h1>
 <p>An error occured while processing your request.</p>
 <h2>Request information</h2>
 <div style="margin-left: 5em;">
 <p>The HTTP request information is included in the following table.</p>
 <table border="1" aria-labelledby="req_info_summary1">
  <caption id="req_info_summary1">HTTP request information</caption>
  <tr><th id="outer_item">Item</th><th id="outer_data">Data</th></tr>
  <tr><td headers="outer_item">Method</td><td headers="outer_data">POST</td></tr>
  <tr><td headers="outer_item">URI</td headers="outer_data"><td headers="outer_data">REMOVED</td></tr>
  <tr><td headers="outer_item"">HTTP Headers</td><td headers="outer_data">
   <table width="90%" aria-labelledby="req_header_summary">
    <caption id="req_header_summary">HTTP request headers</caption>
    <tr><th id="header_key"><small>Key</small></th><th id="header_value"><small>Value</small></th></tr>
    <tr><td headers="header_key"><small>Host</small></td><td headers="header_value"><small>localhost</small></td></tr>
   </table>
  </td></tr>
 </table>
 </div>
</div>
</main>
</body>
</html>
```

2. Open the Recommendations.

## Additional information

- ![screen shot](https://user-images.githubusercontent.com/39089094/107878114-27d28a00-6ed9-11eb-9e6e-37bd9abbe2a3.png)

- Notes
  - It seems to be an issue with nested tables. If the inner table is moved outside of the outer table, no recommendation is issued.

**Comment by tombrunet** [comment]
This is working as designed. `aria-labelledby` and `aria-describedby` are not the same thing. They are exposed by screen readers in different ways. For the purposes of a table summary, `aria-describedby` is expected. You used `aria-labelledby`. 

Labels tend to be short names for controls (labels for inputs, alt of images, etc). Descriptions are for longer summaries. Note also that this rule is a recommendation and not a violation.
